### DISCUSSION TITLE
Feature Request: Add a step at the end of each task that checks for placeholders

### DISCUSSION THREAD

**Discussion by TechMitten** [cline/cline#464] [Feature Requests]
Please add a step to the end of each task that checks each modified file for placeholders and then restores the missing code if it detects placeholders. Either that, or there needs to be a button other than "Reject" that tells the LLM that it used placeholders instead of code and it needs to restore those placeholders with the original code. 

@saoudrizwan This would make claude-dev usable for larger projects. Your efforts are truly appreciated, thank you.

**Comment by arwpc** [comment]
Second this. I have reminders in both my config in Claude.ai and here in the VS Code extension, but it forgets and I continually get placeholders or snippets. I keep blowing away huge sections of my code when it includes a placeholder, which is frequent. Here's an example:

 SNIPPETS OR PLACEHOLDERS. YOU just about deleted 90% of scenePlayerController.js

// ... (keep all existing code up to the executeMotor function)

THANK YOU THANK YOU!!!

**Comment by arwpc** [comment]
![image](https://github.com/user-attachments/assets/47eec55f-88e9-4688-b1e4-4fe95896fdd5)
![snippet](https://github.com/user-attachments/assets/eed959a2-aaa0-49de-8325-f7685b641b4a)

**Comment by arwpc** [comment]
That's what it looks like in vscode when it tries to destroy my code with a snippet. (you can see it in green, and see the red that it's trying to delete)

**Comment by arwpc** [comment]
Here's a new snippet: /* ... (keep all existing styles) ... */

BTW - I've run into this fifteen times yet tonight....PITA
